site/src/sections/app.style.js

```diff
@@ -53,7 +53,7 @@ h4,
 h5,
 h6 {
     font-family: 'Qanelas Soft', 'Open Sans', sans-serif;
-    color: ${(props) => props.theme.tertiaryColor};
+    color: ${(props) => props.theme.white};
```

Comment:
Don't update the global style. Override the change in the respective component only.

Reply:
Sure @Nikhil-Ladha